runtime/onert/core/include/backend/basic/BackendContextHelpers.h

```diff
@@ -219,25 +243,44 @@ template <typename T_BackendContext> ITensorRegistry *genTensors(T_BackendContex
 inline void initConsts(const ir::Operands &operands,
```

Comment:
I prefer this function to remain solely responsible for initializing constant tensors. Could you implement another function for initializing nonconstant tensors of sharing memory?

Reply:
applied ;)